app/components/NextEventInfo/NextEventInfo.test.tsx

```diff
@@ -0,0 +1,70 @@
+import "@testing-library/jest-dom";
+import { render, screen } from "@testing-library/react";
+
+import NextEventInfo from "./NextEventInfo";
+import type { MeetupEvent } from "~/models/meetup";
+
+const DEFAULT_ONLINE_EVENT: MeetupEvent = {
+  title: "Title",
+  shortUrl: "https://short.url",
+  venue: {
+    name: "Online",
+    address: "",
+    city: "",
+    state: "",
+  },
+  dateTime: "2022-01-01T01:00:00Z",
+  going: 100,
+};
+
+describe("NextEventInfo", () => {
+  it("renders given venue", () => {
+    const eventWithVenue = {
+      ...DEFAULT_ONLINE_EVENT,
+      venue: {
+        name: "Venue name",
+        address: "Venue address",
+        city: "Venue city",
+        state: "Venue state",
+      },
+    };
+    render(<NextEventInfo event={eventWithVenue} />);
+    expect(screen.getByTestId("venue-name")).toHaveTextContent("Venue name");
+    expect(screen.getByTestId("venue-location")).toHaveTextContent(
+      "Venue address, Venue city, Venue state"
+    );
+  });
+
+  it("renders default venue for online events", () => {
+    render(<NextEventInfo event={DEFAULT_ONLINE_EVENT} />);
+    expect(screen.getByTestId("venue-name")).toHaveTextContent(
+      "H-E-B Digital & Favor Eastside Tech Hub (and online)"
+    );
+    expect(screen.getByTestId("venue-location")).toHaveTextContent(
+      "2416 E 6th St, Austin, TX"
+    );
+  });
+
+  it("renders date in the correct time zone", () => {
+    render(<NextEventInfo event={DEFAULT_ONLINE_EVENT} />);
+    expect(screen.getByTestId("time")).toHaveTextContent(
+      "Fri. Dec 31, 7:00 PM CST"
+    );
+  });
+
+  it("renders button label when many are attending", () => {
+    const eventWithManyAttending = { ...DEFAULT_ONLINE_EVENT, going: 10 };
+    render(<NextEventInfo event={eventWithManyAttending} />);
+    expect(screen.getByTestId("meetup-button-text")).toHaveTextContent(
+      "Join 10 others at our next meetup!"
+    );
```

Comment:
Do you mind trying to use test ID's a little less? You can achieve the same thing and even add a little bit more value to the test with a `getByRole` query:

```ts
screen.getByRole("link", { name: "Join 10 others at our next meetup!" })
```

The creator of the [testing-library](https://testing-library.com/) framework we're using (on top of vitest) has some good advice/reasoning behind this: https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-querying-by-text

In short:
- Less setup/maintenance keeping track of all the test IDs
- Better accessibility testing when querying by roles/text